src/index.js

```diff
@@ -1,19 +1,60 @@
-const addDebetInput = document.getElementById("add-debet-input");
-addDebetInput.addEventListener("click", addDebetLine);
-const addCreditInput = document.getElementById("add-credit-input");
-addCreditInput.addEventListener("click", addCreditLine);
+//this code delates inputs
+function deliteInput(fun) {
+  fun.addEventListener("click", (e) => {
+    fun.parentElement.remove();
+  });
+}
+//get debet input
+let buttonDebet = document.querySelector("#add-debet-input");
+buttonDebet.addEventListener("click", addDebetLine);
+
+let div = document.querySelector("#Debet-input-area");
```

Comment:
Give more valueable name for the variable. Also looks like it can be replaced with const